lib/util/http/auth.spec.ts

```diff
@@ -110,32 +110,57 @@ describe(getName(), () => {
         }
       `);
     });
-  });
 
-  it(`npm basic token`, () => {
```

Comment:
Looks like wrong indentation?

Reply:
I actually corrected it - it was outside of the `describe` block that described `applyAuthorization`.